Week9Serialization/Week9Serialization/Functional Classes/Person.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Week9Serialization
+{
+
+    /// <summary>
+    /// This is the base class for both the "Male" and "Female" classes.
+    /// </summary>
+    [Serializable]
+    [DataContract]
+    public class Person
+    {
+        /// <summary>
+        /// This method allows the name property to be set on construction
+        /// </summary>
+        /// <param name="name"></param>
+        public Person(string name = null)
+        {
+            Name = name;
+        }
+
+        [DataMember(Order = 2)]
+        public string Name { get; set; }
```

Comment:
Your properties should have XML comments as well. The XML comment convention for properties is something like, "Gets or sets the name for the `<see cref="Person"/>`. For read-only properties, it would be just "Gets the name..." (-0.4)